src/demo/data.js

```diff
@@ -2,51 +2,153 @@ import faker from 'faker/locale/en';
-  const courseEnd = faker.date.future(null, courseStart);
-  const hasPassed = Math.random() <= percentPassed;
-  const grade = hasPassed ? faker.random.number({ min: 70, max: 100 }) :
-    faker.random.number({ max: 70 });
-
-  // TODO: we could generate all columns so we could easily fake the csv call as well
+const numPassed = 120;
+
+const getEnrollments = () => {
+  const fromPastWeekDate = new Date();
+  const fromPastMonthDate = new Date();
+  const toPastMonthDate = new Date();
+  fromPastWeekDate.setDate(fromPastWeekDate.getDate() - 7);
+  fromPastMonthDate.setDate(fromPastMonthDate.getDate() - 30);
+  toPastMonthDate.setDate(toPastMonthDate.getDate() - 60);
+  const dateInPastMonths = faker.date.between(fromPastMonthDate, toPastMonthDate);
+  const dateInPastMonth = faker.date.between(fromPastWeekDate, fromPastMonthDate);
+
+  let enrollmentsData = [...Array(numEnrollments)].map((_, idx) => {
+    // Although we don't pass first & lastname, this makes our emails more consistent
+    const firstName = faker.name.firstName();
+    const lastName = faker.name.lastName();
+    const provider = 'bestrun.com';
+
+    const course = courses[faker.random.number({ max: courses.length - 1 })];
+    const courseStart = faker.date.past();
+    const courseEnd = faker.date.future();
+    const hasPassed = idx < numPassed;
+    const grade = hasPassed ? faker.random.number({ min: 70, max: 100 }) :
+      faker.random.number({ max: 70 });
+    const lastActivityDate = hasPassed ? faker.date.recent(7) : dateInPastMonths;
+
+    return {
+      id: idx,
+      user_email: faker.internet.email(firstName, lastName, provider),
+      consent_granted: true,
+      course_title: course.title,
+      course_price: course.price,
+      course_start: courseStart,
+      course_end: courseEnd,
+      passed_timestamp: hasPassed ? faker.date.recent(7)
+        : null,
+      has_passed: hasPassed,
+      current_grade: grade / 100,
+      last_activity_date: lastActivityDate,
+    };
+  });
+
+  // Add more records to generate data for different tables.
+  const enrollmentsSample = enrollmentsData.slice(0, 5);
+  enrollmentsData = [
+    ...enrollmentsData,
+    ...enrollmentsSample.map(enrollment => ({ ...enrollment, last_activity_date: dateInPastMonth })), // eslint-disable-line max-len
+    ...enrollmentsSample.map(enrollment => ({ ...enrollment, consent_granted: false })), // eslint-disable-line max-len
+    ...enrollmentsSample.map(enrollment => ({ ...enrollment, passed_timestamp: dateInPastMonths, last_activity_date: dateInPastMonths })), // eslint-disable-line max-len
+    ...enrollmentsSample.map(enrollment => ({ ...enrollment, course_end: dateInPastMonth })), // eslint-disable-line max-len
+  ];
+  return enrollmentsData;
+};
+
+const enrollments = getEnrollments();
+
+const filterEnrolledLearners = () => enrollments.filter(enrollment => enrollment.consent_granted === true).map((enrollment) => { // eslint-disable-line max-len
```

Comment:
Can this line be formatted to avoid using `// eslint-disable-line max-len`? It would probably also make it easier to read.